bip-0008.mediawiki

```diff
@@ -16,13 +16,13 @@
 
 This document specifies an alternative to [[bip-0009.mediawiki|BIP9]] that corrects for a number of perceived mistakes.
 Block heights are used for start and timeout rather than POSIX timestamps.
-It additionally introduces an additional activation parameter to guarantee activation of backward-compatible changes (further called "soft forks").
+It additionally introduces an optional activation parameter to guarantee activation of backward-compatible changes (further called "soft forks").
```

Comment:
The parameter isn't optional. What you set the parameter to is optional. I would rather:

"It additionally introduces an ~additional~ activation parameter that **can** guarantee activation of backward-compatible changes"

Reply:
Thanks, added!